x-pack/plugins/security_solution/public/timelines/components/timeline/body/index.test.tsx

```diff
@@ -148,6 +149,19 @@ describe('Body', () => {
           .exists()
       ).toEqual(true);
     });
+    describe('when there is a graphEventId', () => {
+      beforeEach(() => {
+        props.graphEventId = 'graphEventId'; // any string w/ length > 0 works
+      });
+      it('should not render the timeline body', () => {
+        const wrapper = mount(
+          <TestProviders>
+            <Body {...props} />
+          </TestProviders>
+        );
+        expect(wrapper.find(TimelineBody).props().visible).toBe(false);
```

Comment:
the timeline body still renders, but it gets a `display: none` style via `styled-components`.

Reply:
Passing the component (function) reference here. Seems neat. Is this a good thing to do in enzyme? ty

Reply:
Passing the component reference is a totally valid thing to do in Enzyme, however we generally prefer to instrument with `data-test-subj`s, and them for selection in Enzyme tests.

The rational for standardizing on `data-test-subj` means we can instrument the code under test "once", and then use the same test id in both enyzme and functional / Cypress tests.

Consider replacing the use of `TimelineBody` for selection with a `data-test-subj`.

Reply:
thanks for the explanation. i took your advice.